packages/rocketchat-livechat/permissions.js

```diff
@@ -10,10 +10,10 @@ Meteor.startup(() => {
 		RocketChat.models.Roles.createOrUpdate('livechat-guest');
 	}
 	if (RocketChat.models && RocketChat.models.Permissions) {
-		RocketChat.models.Permissions.createOrUpdate('view-l-room', ['livechat-agent', 'livechat-manager', 'admin']);
+		RocketChat.models.Permissions.createOrUpdate('view-l-room', ['bot', 'livechat-agent', 'livechat-manager', 'admin']);
```

Comment:
if the bot will also be a livechat agent, there is no need to give all bots this permission.

this would be useful if you to **all** bots being able to see (or close) livechats _without_ being livechat agents.